test/e2e/kubevirt.go

```diff
@@ -1474,6 +1540,13 @@ passwd:
 				return primaryUDNValueFor("device", field)
 			}
 		)
+		AfterAll(func() {
+			nodesList, err := fr.ClientSet.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			for _, node := range nodesList.Items {
+				Expect(removeImagesInNode(node.Name, strings.Split(kubevirt.FakeLauncherImage, ":")[0])).To(Succeed())
```

Comment:
Let's move the string.Split inside the function

Reply:
I checked and removing the tag is idempotent so I have no issues with adding it.